tests/boulder-integration.sh

```diff
@@ -119,6 +119,8 @@ common revoke --cert-path "$root/conf/live/le1.wtf/cert.pem"
 common revoke --cert-path "$root/conf/live/le2.wtf/cert.pem" \
        --key-path "$root/conf/live/le2.wtf/privkey.pem"
 
+common register --deactivate
```

Comment:
> Is there a way around using the default in --non-interactive mode? Should account deactivation not default to False?

There should definitely be a way for users who run certbot with `--non-interactive` to deactivate their account. The two ways we've solved this problem in the past are to either change the default or make the user provide an extra flag like `--mass-revocation-ok`. I think I prefer just changing the default, but either option (or potentially another approach you can think of) is fine with me.